### ISSUE TITLE
Fine tuning a pre-trained model?

### ISSUE THREAD

**Issue by tjasmin111** [ultralytics/ultralytics#6165]
### Question

So far, we were using the notation of `model = yolov8n.pt` to train a model. Assume I already have a nano model trained on my dataset. Can I fine-tune it on my newer dataset instead of starting from scratch on a yolov8n? So basically using this instead:

`yolo train data=data.yaml model=myPretrained_bestv8n.pt epochs=100`

**Comment by Kayzwer** [comment]
yes you can, and also remember to set warmup epochs to 0, so the learning rate is not set to very high during the first few epochs.

**Comment by tjasmin111** [comment]
Hmm interesting! Can you give more context and examples how to do that? It's new to me!

**Comment by Kayzwer** [comment]
`yolo train data=data.yaml model=yolov8n.pt epochs=100 warmup_epochs=0`

**Comment by glenn-jocher** [comment]
@Kayzwer absolutely, you're on the right track! To fine-tune your pre-trained model on a new dataset, you can use the command you’ve mentioned. Just replace `yolov8n.pt` with the path to your custom pre-trained model file, for instance, `myPretrained_bestv8n.pt`. Setting `warmup_epochs=0` is advised as it prevents the learning rate from starting too high during the initial epochs, which is especially important when fine-tuning because your model has already learned some features from the previous dataset.

Ensure your `data.yaml` is correctly configured for your new dataset. The training will continue from the weights you have provided, adjusting to the nuances of your new data. This process can lead to better performance on the new dataset, especially if it’s similar to the dataset used for the initial training. Good luck with your fine-tuning!
